java/ql/test/library-tests/frameworks/JaxWs/JaxRsFlow.ql

```diff
@@ -27,28 +27,12 @@ class ValueFlowConf extends DataFlow::Configuration {
     exists(MethodAccess ma | ma.getMethod().hasName("sink") | n.asExpr() = ma.getAnArgument())
   }
 
+  // TODO: move to default?
```

Comment:
Good point. Tests are usually not that big, so I don't think this would normally cause performance problems, and it would prevent the broken flows issue by default.